config/config_mgmt.py

```diff
@@ -584,7 +593,125 @@ def testRun_Delete_Add_Ports():
 
     return
 
-if __name__ == '__main__':
+def testRun_Delete_Add_Ports_2():
+
+    #TODO: Verify config in Config DB  after writing to config DB.
+    print('Test Run Delete Ports')
+    try:
+        cm = configMgmt('configDB', debug=True)
+    except Exception as e:
+        print(e)
+        return
+
+    deps, ret = cm.deletePorts(delPorts=['Ethernet0', 'Ethernet2'], force=True)
+    if ret == False:
+        print("Port Deletion Test failed")
+        return None
+
+    print("\n***Port Deletion Test Passed***\n")
+    import time
+    time.sleep(5)
+
+    portJson = {
+        "PORT": {
+            "Ethernet0": {
+                    "alias": "Eth1/1",
+                    "admin_status": "up",
+                    "lanes": "65,66,67,68",
+                    "description": "",
+                    "speed": "100000"
+            }
+        }
+    }
+
+    #ret = cm.addPorts(ports=['Ethernet0'], portJson=portJson, loadDefConfig=False)
+    ret = cm.addPorts(ports=['Ethernet0'], portJson=portJson, loadDefConfig=True)
```

Comment:
configuration is defined in portJson,   should the loadDefConfig be False?

Reply:
portJson contains only PORT Table Config.